src/control/ros/RosPositionCommandExecutor.cpp

```diff
@@ -0,0 +1,139 @@
+#include <aikido/control/ros/RosPositionCommandExecutor.hpp>
+#include <aikido/control/ros/Conversions.hpp>
+#include <aikido/control/ros/util.hpp>
+
+namespace aikido {
+namespace control {
+namespace ros {
+
+using std::chrono::milliseconds;
+
+//=============================================================================
+RosPositionCommandExecutor::RosPositionCommandExecutor(
+      ::ros::NodeHandle node,
+      const std::string& serverName,
+      const std::vector<std::string>& jointNames,
+      const std::chrono::milliseconds connectionTimeout,
+      const std::chrono::milliseconds connectionPollingPeriod)
+  : mNode(std::move(node))
+  , mCallbackQueue{}
+  , mClient{mNode, serverName, &mCallbackQueue}
+  , mJointNames{jointNames}
+  , mConnectionTimeout{connectionTimeout}
+  , mConnectionPollingPeriod{connectionPollingPeriod}
+  , mInProgress{false}
+{
+}
+
+//=============================================================================
+RosPositionCommandExecutor::~RosPositionCommandExecutor()
+{
+  // Do nothing.
+  // TODO: Should we wait for the current trajectory to finish executing?
```

Comment:
Nit: This comment was copied and pasted from `RosTrajectoryExecutor`. There are no trajectories in this file! 😁